R/chembl.R

```diff
@@ -0,0 +1,226 @@
+#' Query ChEMBL using ChEMBL IDs
+#'
+#' Retrieve ChEMBL data using a vector of ChEMBL IDs.
+#' @param query character; a vector of ChEMBL IDs.
+#' @param resource character; the ChEMBL resource to query. Use
+#' \code{chembl_resources()} to see all available resources.
```

Comment:
link to chembl_resources() help page
```suggestion
#' [chembl_resources()] to see all available resources.
```